local-volume/provisioner/pkg/discovery/discovery.go

```diff
@@ -179,25 +179,24 @@ func (d *Discoverer) createPV(file, class string, config common.MountConfig, cap
 	glog.Infof("Created PV %q for volume at %q", pvName, outsidePath)
 }
 
-// roundDown the capacity so that it is easier to read
+// roundDown the capacity to an easy to read value
 func roundDownCapacityPretty(capacity int64) int64 {
-	// capacity is exactly divisible by Gi
-	if capacity%1024*1024*1024 == 0 {
-		return capacity
-	}
-	sizeGi := util.RoundUpSize(capacity, 1024*1024*1024)
-	if (sizeGi - 1) >= 10 {
-		// rounddown to nearest Gi
-		return (sizeGi - 1) * 1024 * 1024 * 1024
-	}
-	// capacity is exactly divisible by Mi
-	if capacity%1024*1024 == 0 {
-		return capacity
-	}
-	sizeMi := util.RoundUpSize(capacity, 1024*1024)
-	if (sizeMi - 1) >= 10 {
-		// rounddown to nearest Mi
-		return (sizeMi - 1) * 1024 * 1024
+
+	const (
+		KiB int64 = 1024
+		MiB int64 = 1024 * KiB
+		GiB int64 = 1024 * MiB
+		TiB int64 = 1024 * GiB
+	)
+	easyToReadSizes := []int64{TiB, GiB, MiB}
```

Comment:
I think rounding down a whole TB might be too much

Reply:
This would only happen for drives with more than 10TB. If I don't hear back from you, I'll assume its too much.

Reply:
Yeah, let's keep the granularity at GB and MB.

Reply:
Done.